changelog_generator.php

```diff
@@ -82,7 +82,7 @@
 
 $client->setUri(
     'https://api.github.com/search/issues?q=' . urlencode(
-        'milestone:' . $milestonePayload['title']
+        'milestone:"' . $milestonePayload['title'] . '"'
```

Comment:
Good catch, I've added a straightforward replacement of `"` to `\"` which is all that is required. I looked into using `addslashes()` but it is only `"` that must be escaped (and escaping others such as ``'` will result in the search failing.